HLTrigger/Configuration/python/deepTauAtHLT.py

```diff
@@ -0,0 +1,357 @@
+import FWCore.ParameterSet.Config as cms
+
+from RecoTauTag.RecoTau.pfRecoTauDiscriminationByIsolation_cfi import *
+from RecoTauTag.RecoTau.PFRecoTauQualityCuts_cfi import PFTauQualityCuts
+
+
+from RecoTauTag.RecoTau.PFTauPrimaryVertexProducer_cfi      import *
+from RecoTauTag.RecoTau.PFTauSecondaryVertexProducer_cfi    import *
+from RecoTauTag.RecoTau.PFTauTransverseImpactParameters_cfi import *
+
+
+def update(process):
+    process.options.wantSummary = cms.untracked.bool(True)
+
+    process.hltFixedGridRhoFastjetAll = cms.EDProducer( "FixedGridRhoProducerFastjet",
+        gridSpacing = cms.double( 0.55 ),
+        maxRapidity = cms.double( 5.0 ),
+        pfCandidatesTag = cms.InputTag( "hltParticleFlowReg" )
+    )
+
+    PFTauQualityCuts.primaryVertexSrc = cms.InputTag("hltPixelVertices")
+
+    process.chargedIsoPtSum = pfRecoTauDiscriminationByIsolation.clone(
+        PFTauProducer = cms.InputTag('hltHpsPFTauProducerReg'),
+        particleFlowSrc = cms.InputTag("hltParticleFlowReg"),
+        vertexSrc = PFTauQualityCuts.primaryVertexSrc,
+        Prediscriminants = cms.PSet(  BooleanOperator = cms.string( "and" ) ),
+        ApplyDiscriminationByECALIsolation = cms.bool(False),
```

Comment:
In general, please don't redefine type of config arguments for basic types like cms.bool, cms.double... For options that are already set in the cloned object it is enough and preferred by RECO to set the type (`... = False,`).